insights/parsers/tests/test_chkconfig.py

```diff
@@ -61,14 +61,14 @@ def test_chkconfig():
     assert not chkconfig.is_on('rlogin')
 
 
-def test_command_not_found():
-    context = context_wrap(ERROR)
-    chkconfig = ChkConfig(context)
-    assert len(chkconfig.services) == 0
-    assert len(chkconfig.parsed_lines) == 0
-    assert not chkconfig.is_on('crond')
-    assert not chkconfig.is_on('kdump')
-    assert not chkconfig.is_on('restorecond')
+# def test_command_not_found():
+#     context = context_wrap(ERROR)
+#     chkconfig = ChkConfig(context)
+#     assert len(chkconfig.services) == 0
+#     assert len(chkconfig.parsed_lines) == 0
+#     assert not chkconfig.is_on('crond')
+#     assert not chkconfig.is_on('kdump')
+#     assert not chkconfig.is_on('restorecond')
```

Comment:
Parsers should not break backward backward compatibility so you may need to look at each of these parsers to see what they returned when the bad words were present in the input data, if they were not throwing an exception. (if you haven't already done that.)

Reply:
The parser (parse_context()) should never get called, if a `bad_line` is found, since the CommandParser is catching the issue before parser_context is called and is raising a ContentException (which is based on SkipComponent).